bool reset
float32 cbf_param
---
bool done
